Source/SwiftLintFramework/Rules/Lint/UnusedDeclarationRule.swift

```diff
@@ -171,8 +171,7 @@ private extension SwiftLintFile {
 
         let cursorInfo = self.cursorInfo(at: nameOffset, compilerArguments: compilerArguments)
 
-        if let annotatedDecl = cursorInfo?.annotatedDeclaration,
-            ["@IBAction", "@objc"].contains(where: annotatedDecl.contains) {
```

Comment:
I'm surprised the IBAction isn't required here still, but the tests cover it so nbd if they pass!

Reply:
It's already covered by `declarationAttributesToSkip`:

https://github.com/realm/SwiftLint/blob/fd97e2259c885aee7d8ec3120c35de57fde592f5/Source/SwiftLintFramework/Rules/Lint/UnusedDeclarationRule.swift#L324-L330

Reply:
ah sorry I was misremembering this change https://github.com/realm/SwiftLint/pull/3458